tests/themes/test_themes.js

```diff
@@ -5,24 +5,22 @@
  */
 'use strict';
 
-goog.provide('Blockly.TestThemes');
-
 
 /**
  * A theme with classic colours but enables start hats.
  */
-Blockly.Themes.TestHats = Blockly.Theme.defineTheme('testhats', {
+const TestHatsTheme = Blockly.Theme.defineTheme('testhats', {
   'base': Blockly.Themes.Classic
 });
```

Comment:
This has side effects right? And that's how we can access the theme later? Just want to clarify.

Reply:
Yes, this has side effects. We register themes in their constructors. However, I believe in this case the `populateTestThemes` method is how we are adding the themes to the advanced playground.